menu/Menu.java

```diff
@@ -0,0 +1,229 @@
+        labelNome.setCursor(new java.awt.Cursor(java.awt.Cursor.DEFAULT_CURSOR));
+
+        /* Configuração do campo de nome */
+        inputNome.setFont(new java.awt.Font("Segoe UI", 1, 32)); // NOI18N
+        inputNome.setHorizontalAlignment(javax.swing.JTextField.CENTER);
+        inputNome.setAutoscrolls(false);
+        inputNome.setBorder(new javax.swing.border.SoftBevelBorder(javax.swing.border.BevelBorder.LOWERED, new java.awt.Color(204, 204, 204), new java.awt.Color(204, 204, 204), new java.awt.Color(204, 204, 204), new java.awt.Color(204, 204, 204)));
+        inputNome.setDisabledTextColor(new java.awt.Color(255, 255, 255));
+        inputNome.setMargin(new java.awt.Insets(3, 6, 3, 6));
+        inputNome.setSelectionColor(new java.awt.Color(51, 51, 51));
+        inputNome.addActionListener(new java.awt.event.ActionListener() {
+            public void actionPerformed(java.awt.event.ActionEvent evt) {
+                inputNomeActionPerformed(evt);
+            }
+        });
+
+        /* Configuração do botão de iniciar */
+        iniciar.setForeground(new java.awt.Color(255, 255, 255));
+        iniciar.setText("Iniciar");
+        iniciar.setBorderColor(new java.awt.Color(242, 242, 242));
+        iniciar.setBorderPainted(false);
+        iniciar.setColor(new java.awt.Color(209,139,71));
+        iniciar.setColorClick(new java.awt.Color(209,139,71));
+        iniciar.setColorOver(new java.awt.Color(255,206,158));
+        iniciar.setFont(new java.awt.Font("Consolas", 1, 24)); // NOI18N
+        iniciar.setRadius(10);
+        iniciar.setVerticalAlignment(javax.swing.SwingConstants.BOTTOM);
+        iniciar.addActionListener(new java.awt.event.ActionListener() {
+            public void actionPerformed(java.awt.event.ActionEvent evt) {
+                iniciarActionPerformed(evt);
+            }
+        });
+
+        /* Configuração do botão de sair */
+        sair.setForeground(new java.awt.Color(255, 255, 255));
+        sair.setText("Sair");
+        sair.setToolTipText("");
+        sair.setBorderColor(new java.awt.Color(242, 242, 242));
+        sair.setBorderPainted(false);
+        sair.setColor(new java.awt.Color(209,139,71));
+        sair.setColorClick(new java.awt.Color(209,139,71));
+        sair.setColorOver(new java.awt.Color(255,206,158));
+        sair.setFont(new java.awt.Font("Consolas", 1, 24)); // NOI18N
+        sair.setHorizontalTextPosition(javax.swing.SwingConstants.CENTER);
+        sair.setRadius(10);
+        sair.setVerticalAlignment(javax.swing.SwingConstants.BOTTOM);
+        sair.addMouseListener(new java.awt.event.MouseAdapter() {
+            public void mouseEntered(java.awt.event.MouseEvent evt) {
+                sairMouseEntered(evt);
+            }
+        });
+        sair.addActionListener(new java.awt.event.ActionListener() {
+            public void actionPerformed(java.awt.event.ActionEvent evt) {
+                sairActionPerformed(evt);
+            }
+        });
+
```

Comment:
Quando for implementar os campos de escolha de cor, de modo de jogo (vs. IA ou PVP) etc você pode implementar seguindo a lógica de feature flags. Para a Demo acho que podemos apresentar o menu só com os campos atuais de menu inicial. O que acha?

Reply:
Ok!